packages/r3f/src/components/editable.tsx

```diff
@@ -140,7 +140,7 @@ const createEditable = <Keys extends keyof JSX.IntrinsicElements>(
             visible={visible !== 'editor' && visible}
             userData={{
               __editable: true,
-              __editableName: objectKey,
+              __editableName: storeKey,
```

Comment:
Probably should rename `__editableName` too, right?

Reply:
True